cmd/minikube/cmd/config/configure.go

```diff
@@ -61,7 +61,7 @@ var addonsConfigureCmd = &cobra.Command{
 				awsAccessKey = AskForStaticValue("-- Enter AWS Secret Access Key: ")
 				awsRegion = AskForStaticValue("-- Enter AWS Region: ")
 				awsAccount = AskForStaticValue("-- Enter 12 digit AWS Account ID: ")
-				awsRole = AskForStaticValue("-- (Optional) Enter ARN of AWS role to assume: ")
+				awsRole = AskForStaticValue("-- (Optional) Enter ARN of AWS role to assume: ", true)
```

Comment:
Sorry it took so long to review.

I personally don't really care about adding another small function here.
I think it would be simplest to just add a AskForStaticValueOptional, which is the same as AskForStaticValue without the loop and length check.

To avoid the little duplication, you could even have a func getStaticValue which is called by both.  AskForStaticValueOptional returns this function directly, which AskForStaticValue does the length check and loop

Reply:
@r2d4 thank you for you advice, please check again.